contrib/kubernetes/static/network-ovs.yaml

```diff
@@ -0,0 +1,61 @@
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: ovs
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the openvswitch daemon.
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        component: network
+        type: infra
+        openshift.io/role: network
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      # Requires fairly broad permissions - ability to read all services and network functions as well
+      # as all pods.
```

Comment:
This comment isn't accurate for the OVS pod; it shouldn't need any kubernetes access at all

Reply:
dns and kube-proxy are in there too.

Reply:
I saw something reading pods, will double check who it was.

Reply:
No, I mean, that comment is correct where it appears in network-daemonset.yaml, but I was pointing out that the same comment also appears in network-ovs.yaml, where it's totally wrong.

Reply:
Ok, thanks.